card/spec/lib/card/reference_spec.rb

```diff
@@ -110,81 +110,81 @@
 
   it "update referencing content on rename junction card" do
     @ab = Card["A+B"] #linked to from X, included by Y
-    @ab.update_attributes! :name=>'Peanut+Butter', :update_referencers => true
+    @ab.update_attributes! name: 'Peanut+Butter', update_referencers: true
     @x = Card['X']
     expect(@x.content).to eq("[[A]] [[Peanut+Butter]] [[T]]")
   end
 
   it "update referencing content on rename junction card" do
     @ab = Card["A+B"] #linked to from X, included by Y
-    @ab.update_attributes! :name=>'Peanut+Butter', :update_referencers=>false
+    @ab.update_attributes! name: 'Peanut+Butter', update_referencers: false
     @x = Card['X']
     expect(@x.content).to eq("[[A]] [[A+B]] [[T]]")
   end
 
   it "template inclusion" do
-    cardtype = Card.create! :name=>"ColorType", :type=>'Cardtype', :content=>""
-    Card.create! :name=>"ColorType+*type+*structure", :content=>"{{+rgb}}"
-    green = Card.create! :name=>"green", :type=>'ColorType'
+    Card.create! name: 'ColorType', type: 'Cardtype', content: ''
+    Card.create! name: 'ColorType+*type+*structure', content: '{{+rgb}}'
+    green = Card.create! name: "green", type: 'ColorType'
     rgb = newcard 'rgb'
-    green_rgb = Card.create! :name => "green+rgb", :content=>"#00ff00"
+    green_rgb = Card.create! name: "green+rgb", content: "#00ff00"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.